src/Type/UnionIterableType.php

```diff
@@ -2,14 +2,18 @@
 
 namespace PHPStan\Type;
 
-class UnionIterableType implements IterableType, UnionType
+class UnionIterableType implements UnionType
 {
 
 	use IterableTypeTrait;
 
 	/** @var \PHPStan\Type\Type[] */
 	private $types;
 
+	/**
+	 * @param Type   $itemType
```

Comment:
Please do not align phpDocs like this, it creates extra lines in diffs when it has to be realigned...